map/map_loader/script/map_hash_generator

```diff
@@ -54,9 +58,30 @@ class MapHashGenerator(Node):
         return path.read_text() if path.is_file() else ""
 
     @staticmethod
-    def generate_file_hash(data):
+    def generate_lanelet_file_hash(data):
         return hashlib.sha256(data.encode()).hexdigest() if data else ""
 
+    @staticmethod
+    def generate_pcd_file_hash(path):
+        path = pathlib.Path(path)
+        if path.is_dir():
+            pcd_file_paths = [f for f in path.iterdir() if pathlib.Path(f).suffix == ".pcd"]
+        elif path.is_file():
+            if path.suffix == ".pcd":
+                pcd_file_paths = [path]
+        else:
+            return ""
+
+        concatenated_hash = ""
+        for pcd_file_path in pcd_file_paths:
```

Comment:
The paths from iterdir is in arbitrary order, so they have to be sorted.

Reply:
@isamu-takagi fixed it in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/745/commits/f08542fb191cd11df2b09b38bd300e20c3ad50ff